src/main/resources/META-INF/resources/primefaces-extensions/counter/1-counter-widget.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * PrimeFaces Extensions Counter Widget
+ * 
+ */
+PrimeFaces.widget.ExtCounter = PrimeFaces.widget.BaseWidget.extend({
+
+    /**
+     * Initializes the widget.
+     * 
+     * @param {object}
+     *        cfg The widget configuration.
+     */
+    init: function (cfg) {
+
+        this._super(cfg);
+        this.id = cfg.id;
+        this.cfg = cfg;
+
+        this.end = cfg.end;
+        this.autoStart = cfg.autoStart;
+        this.oncountercomplete = cfg.oncountercomplete;
```

Comment:
Need to wrap this oncountercomplete so you can call `this.callBehavior('end');` whether the user passes in a custom function or now

Reply:
I did not understand this :(

Reply:
I can take care of this one don't worry about it.